documentation/importing-dependencies/tutorial-ways-of-importing.asciidoc

```diff
@@ -88,7 +102,14 @@ structure is created for the `MainLayout` component regardless of availability o
   }
 }
 ----
++
+** In the examples, `/html/layout.html` is loaded and injected before creating the client-side structure for the `MainLayout` component, regardless of the availability of the `/js/animation.js` script.
 
-[NOTE]
-See corresponding tutorials <<tutorial-include-css#,Including Style Sheets>> and <<tutorial-importing#,Importing html/javascript>>
-about static resource paths and URLs.
+// What about the html vs css load order?
+
+* Dependencies with the same load mode are guaranteed to load in the order defined in the component. This is true for all load modes.
+
+* No guarantees are made on the order of dependencies between groups. For example, if your component has both lazy and inline dependencies, both groups of dependencies will be loaded simultaneously. 
+
+// Doesn't this last statement contradict what is stated in the inline and lazy explanations above? Thoese imply that inline is loaded before lazy?? 
```

Comment:
That is quite odd statement indeed. I'll ask this and the next thing from our experts before landing this patch.

Reply:
Created a separate issue about this as nobody from the development team don't seem to be able to answer. I'll remove the controversial part and land the patch otherwise to get forward.